src/item.cpp

```diff
@@ -2592,33 +2592,113 @@ void item::armor_info( std::vector<iteminfo> &info, const iteminfo_query *parts,
         std::string format;
         if( has_flag( flag_FIT ) ) {
             format = _( "<num> <info>(fits)</info>" );
-        } else if( has_flag( flag_VARSIZE ) && encumbrance ) {
+        } else if( has_flag( flag_VARSIZE ) ) {
             format = _( "<num> <bad>(poor fit)</bad>" );
         }
 
         //If we have the wrong size, we do not fit so alert the player
-        if( sizing_level == sizing::human_sized_small_char )  {
+        if( sizing_level == sizing::human_sized_small_char ) {
             format = _( "<num> <bad>(too big)</bad>" );
         } else if( sizing_level == sizing::big_sized_small_char ) {
             format = _( "<num> <bad>(huge!)</bad>" );
         } else if( sizing_level == sizing::small_sized_human_char ||
-                   sizing_level == sizing::human_sized_big_char )  {
+                   sizing_level == sizing::human_sized_big_char ) {
             format = _( "<num> <bad>(too small)</bad>" );
-        } else if( sizing_level == sizing::small_sized_big_char )  {
+        } else if( sizing_level == sizing::small_sized_big_char ) {
             format = _( "<num> <bad>(tiny!)</bad>" );
         }
 
-        info.push_back( iteminfo( "ARMOR", _( "<bold>Encumbrance</bold>: " ), format,
-                                  iteminfo::no_newline | iteminfo::lower_is_better,
-                                  encumbrance ) );
+        info.push_back( iteminfo( "ARMOR", _( "<bold>Encumbrance</bold>:" ), format,
+                                  iteminfo::lower_is_better ) );
 
         if( const islot_armor *t = find_armor_data() ) {
-            if( t->max_encumber != t->encumber ) {
-                const int encumbrance_when_full =
-                    get_encumber( g->u, encumber_flags::assume_full );
-                info.push_back( iteminfo( "ARMOR", space + _( "Encumbrance when full: " ), "",
-                                          iteminfo::no_newline | iteminfo::lower_is_better,
-                                          encumbrance_when_full ) );
+            if( !t->data.empty() ) {
+                std::unordered_map<std::string, std::tuple<int, int, int>> encumb_data;
+
+                for( const auto &piece : t->data ) {
+                    if( piece.covers.has_value() ) {
+                        for( const auto &covering : piece.covers.value() ) {
+                            if( covering != bodypart_str_id( "num_bp" ) ) {
+                                encumb_data.insert( {
+                                    covering.c_str(), {
```

Comment:
This will store the *id*  of the bodypart. This is not a translated string. It will be the same regardless of the players chosen language.